shared/editor/plugins/PlaceholderPlugin.ts

```diff
@@ -0,0 +1,77 @@
+import filter from "lodash/filter";
+import find from "lodash/find";
+import map from "lodash/map";
+import { Node } from "prosemirror-model";
+import { EditorState, Plugin } from "prosemirror-state";
+import { Decoration, DecorationSet } from "prosemirror-view";
+
+type Config = Array<{
+  /** Condition to meet for the placeholder to be applied to a node */
+  cond: (
```

Comment:
```suggestion
  condition: (
```

Lets change this to the full word, we're not short on characters